test/second_q/operators/test_bosonic_op.py

```diff
@@ -198,25 +198,25 @@ def test_index_order(self):
             self.assertEqual(bos_op, targ)
 
         with self.subTest("index order + simplify"):
-            orig = BosonicOp({"+_1 -_0 +_0 -_0": 1, "-_0 +_1": 2})
+            orig = BosonicOp({"+_1 -_0": 1, "-_0 +_1": 2})
             bos_op = orig.index_order().simplify()
```

Comment:
```suggestion
            bos_op = orig.index_order()
```
This is actually no longer needed